.drone.yml

```diff
@@ -380,3 +380,11 @@ name: github_api_token
 ---
 kind: signature
```

Comment:
Signature should be last thing in the file, also it needs to be updated before it will run.  See instructions [here](https://github.com/grafana/oncall/tree/dev/dev#update-drone-build)

Reply:
I've moved the signature section to the end and updated the signature according to the instructions